src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -186,6 +189,38 @@ def sort_code_mapping_configs(
                     sorted_configs.insert(0, config)
         return sorted_configs
 
+    def get_commit_sha(
+        self,
+        integration_installation: GitHubClientMixin,
```

Comment:
This is very provider specific. Ideally there's a parent class that represents all the various source code integration providers we have.